src/uint.rs

```diff
@@ -388,6 +388,265 @@ impl_concat! {
     (U4096, 4096)
 }
 
+#[cfg(feature = "cross-size")]
+impl_concat_cross_sizes! {
+    (U64, 64),
+    (
+    (U128, 128),
+    (U256, 256),
+    (U384, 384),
+    (U512, 512),
+    (U640, 640),
+    (U768, 768),
+    (U896, 896),
+    (U1024, 1024),
+    (U1280, 1280),
+    (U1536, 1536),
+    (U1792, 1792),
+    (U2048, 2048),
+    (U3072, 3072),
+    (U3584, 3584),
+    (U4096, 4096),
+    (U6144, 6144),
+    (U8192, 8192)
+    )
+}
```

Comment:
Can you move these out of `uint.rs` into their own submodule? It makes the uint.rs file huge, and if you move them to their own module, you can feature-gate them in one place instead of every time the macro is invoked

Reply:
Yes, I thought it is better this way anyways, but the original code had it here so I followed through.

Can I put them in concat?

Reply:
Or do you want a seperate "cross-size" module

Reply:
`concat` works, although maybe put them in an inline submodule so you can gate them in a single place, e.g. `mod cross_size`

Reply:
I'm confused what you can't do? Regardless this is hundreds and hundreds of lines of feature-gated macro invocations which should find a better home than the toplevel module.

Reply:
I can create a separate module for all of this then. Will be easier.